ci/do_ci.sh

```diff
@@ -252,6 +260,14 @@ elif [[ "$CI_TARGET" == "bazel.coverity" ]]; then
      "${ENVOY_BUILD_DIR}"/envoy-coverity-output.tgz \
      "${ENVOY_DELIVERY_DIR}"/envoy-coverity-output.tgz
   exit 0
+elif [[ "$CI_TARGET" == "bazel.fuzz" ]]; then
+  setup_clang_toolchain
+  echo "bazel ASAN libFuzzer build with fuzz tests ${FUZZ_TEST_TARGETS}"
+  echo "Building envoy fuzzers and executing 100 fuzz iterations..."
+  for t in ${FUZZ_TEST_TARGETS}; do
+    bazel_with_collection run ${BAZEL_BUILD_OPTIONS} --config=asan-fuzzer $t -- -runs=10
```

Comment:
You should be able to run multiple target at same time, otherwise the CI time will be long, and not utilizing the resources we have in RBE.

Reply:
Seems like build and test can take multiple test targets, but run can't.
Would enough time be saved if I built the targets all at once, but ran them sequentially?

Reply:
Then my next question is: can you make fuzzer target as `cc_test`? `cc_test` is basically same as `cc_binary` with extra env vars, and you can pass args with `--test_arg`? The parallel run will not only save time but also network because testing with RBE doesn't download the binary (which is around 800MB per binary) to test.

Reply:
yes! That worked swimmingly with something like
`bazel test --config=asan-fuzzer //test/server/config_validation:config_fuzz_test_with_libfuzzer --test_arg="-runs=10"`

It seems like it's linking dependencies now with ubsan (so it's crashing on ubsan errors in external libraries). Is this a difference between `cc_binary` and `cc_test` as well?

Reply:
> It seems like it's linking dependencies now with ubsan (so it's crashing on ubsan errors in external libraries). Is this a difference between cc_binary and cc_test as well?

That shouldn't, though I don't know why are you seeing this, can you push your change so I can see what's happening in CI?

Reply:
Sure, the errors look like
`SUMMARY: UndefinedBehaviorSanitizer: undefined-behavior external/com_google_protobuf/src/google/protobuf/map_type_handler.h:636:17`

they should probably be caught by the fuzzer after the 10 iterations in at least some of the fuzz tests.

Reply:
Ah, this is because `UBSAN_OPTIONS` and `ASAN_OPTIONS` are specified as `--test_env` [here](https://github.com/envoyproxy/envoy/blob/master/.bazelrc#L43-L44), so UBSAN failure is doesn't halt when you do `bazel run` but halts when you do `bazel test`, so the failures are legitimate I think? If you want override those options add `--test_env` in asan-fuzzer config.

Reply:
Thank you! Yeah, those are legitimate UBSan faults, but not in our test code.